content/news/2023-05-18-maplibre-newsletter-may-2023/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "MapLibre Newsletter May 2023"
+date: 2023-05-16
+categories: ["newsletter"]
+authors: [duje]
+draft: false
+---
+
+# MapLibre Newsletter - May 2023
+
+## Highlights
+
+We would like to thank our first sponsor [Mierune](https://www.mierune.co.jp/) for their continued trust in MapLibre.
+
+Many real-world applications have been built with MapLibre. As suggested in https://github.com/maplibre/maplibre/discussions/242 it is about bringing the Community together. Implementing new ideas is what will make MapLibre thrive further. We want to recognize all of your efforts, collectively, and so decided to expand on the idea of the [MapLibre Innovation Lab](https://lab.maplibre.org/).
+
+<a href="https://www.ogc.org/initiatives/ogcsprints/">
+<img src="ogc-sprint.jpg">
```

Comment:
Cool picture! Please add a maxwidth or something that it does not show too large. You can check how it was done in other news articles.